pkg/atlas/providers.json

```diff
@@ -0,0 +1,34 @@
+{
+    "providers" : [
+        {
+            "UNSET": {
```

Comment:
I don't really understand the point of the "SET" or "UNSET" objects. If the provider is included within the file then it should be used. If it shouldn't be used then it shouldn't be included within the file.